modules/images/regenerate-existing-images/helper.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Functions used by the background process API.
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+/**
+ * Creates a background job.
+ *
+ * Job refers to the term and queue refers to the `background_job` taxonomy.
+ *
+ * @since n.e.x.t
+ *
+ * @param string $name Name of job identifier.
+ * @param array  $data Data for the job.
```

Comment:
See my other comment, what can be put into `$data`? Are there any predefined keys we expect, or is it entirely arbitrary? If there are certain keys that are supported, we should document those here.

Reply:
@felixarntz `$data` is optional because we have `$name` mandatory in this function. `$data` can be used to store additional information for the job and the manner in which it will get executed, which is not necessarily identical or in particular manner for different jobs.

However; I am updating the data in DB only if it is present as per your suggestion.
CC: @jjgrainger

Reply:
Thanks, makes sense. In that case, I think we should clarify that "anything" can be passed here. Also let's add the "Optional." prefix and default value information commonly used in WordPress docs.
```suggestion
 * @param array  $data Optional. Arbitrary data for the job. Default empty array.
```